mage_ai/settings/server.py

```diff
@@ -73,7 +73,11 @@ def get_bool_value(value: str) -> bool:
 HOSTNAME = os.getenv('HOSTNAME')
 REDIS_URL = os.getenv('REDIS_URL')
 SERVER_VERBOSITY = os.getenv('SERVER_VERBOSITY', 'info') or 'info'
-SERVER_LOGGING_FORMAT = os.getenv('SERVER_LOGGING_FORMAT')
+SERVER_LOGGING_FORMAT = os.getenv('SERVER_LOGGING_FORMAT', 'plaintext')
+SERVER_LOGGING_TEMPLATE = os.getenv(
```

Comment:
can you add this variable to the `MAGE_SETTINGS_ENVIRONMENT_VARIABLES` list at the bottom of this file?

Reply:
Done